docs/tutorials/multimodal/customization.md

```diff
@@ -0,0 +1,424 @@
+# Customize AutoMM
+:label:`sec_automm_customization`
+
+AutoMM has a powerful yet easy-to-use configuration design.
+This tutorial walks you through various AutoMM configurations to empower you the customization flexibility. Specifically, AutoMM configurations consist of four parts: 
+- optimization
```

Comment:
I guess we need to add an empty line above so the list can be appropriately rendered